andela/ah-backend-prime
# #164798303 Pagination support for articles

#### What does this PR do?
- Adds pagination support for articles
#### Description of tasks to be completed.
- Add default pagination class in settings
- Add a custom pagination class
- Add pagination_classes to the view function
#### How should this be manually tested?
- Fetch this branch from remote by running the command git fetch origin ft-pagination-164798303 in the terminal.
- Check into the local branch git checkout ft-pagination-164798303
- Create and activate your virtual environment
- Install requirements/ dependencies by running pip install -r requirements.txt
- Make migrations with command python manage.py makemmigrations
- Run the command python manage.py migrate to migrate
- Source your .env
- Run the application python manage.py runserver
- Then navigate to the endpoint `http://127.0.0.1:8000/api/v1/articles/` using postman and send a `GET` request
#### Relevant pivotal tracker stories
[#164798303](https://www.pivotaltracker.com/story/show/164798303)

#### Screenshots
Pagination with links to previous and next pages
![image](https://user-images.githubusercontent.com/36891299/55956808-81c6f080-5c6d-11e9-9c16-598f9bd2639c.png)



[Finishes #164798303]
